org.eclipse.xtext.tests/src/org/eclipse/xtext/formatting2/internal/FormatterTestLanguageJavaFormatter.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2020 itemis AG (http://www.itemis.eu) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.xtext.formatting2.internal;
+
+import org.eclipse.xtext.formatting2.AbstractJavaFormatter;
+import org.eclipse.xtext.formatting2.IFormattableDocument;
+import org.eclipse.xtext.formatting2.internal.formattertestlanguage.IDList;
+import org.eclipse.xtext.formatting2.internal.services.FormatterTestLanguageGrammarAccess;
+
+import com.google.inject.Inject;
+
+/**
+ * @author Arne Deutsch - Initial contribution and API
+ */
+public class FormatterTestLanguageJavaFormatter extends AbstractJavaFormatter {
+
+	@Inject
+	FormatterTestLanguageGrammarAccess grammar;
+
+	protected void format(IDList list, IFormattableDocument doc) {
+		doc.prepend(regionFor(list).keyword("idlist"), it -> it.noSpace());
```

Comment:
I know it's a test, but for documentation purpose it would be good to find a better name for `it`

Reply:
`it -> it.` is clutter that I can not get rid of. `noSpace()` only, as in Xtend, is much more nice. I can not make a nice name of it, just want to keep it short. This is why I stick to the "default" ... other than that I would choose a one-letter-name ...

Reply:
f -> f is also nonsaying

IHiddenRegionFormatter::noSpace

is also lengthy